packages/website/pages/account/payment.js

```diff
@@ -67,7 +68,7 @@ const PaymentSettingsPage = props => {
       setIsFetchingPaymentSettings(true);
       setNeedsFetchPaymentSettings(false);
       try {
-        setPaymentSettings(await userBillingSettings());
+        setPaymentSettings(await userBillingSettings(constants.TERMS_OF_SERVICE_VERSION));
```

Comment:
```suggestion
        setPaymentSettings(await userBillingSettings());
```

I dont think we should need to pass an agreement value just to GET the payment settings.

Reply:
I did it